v2/thirdparty/common-customizations/sign-in-and-up/provider-config.mdx

```diff
@@ -3566,6 +3566,10 @@ curl --location --request PUT '^{coreInjector_uri_without_quotes}/<TENANT_ID>/re
 
 ## SAML login
 
+To configure SAML login with SuperTokens, ensure that you use the correct provider name in the third-party config.
+You can find the list of supported SAML providers [here](https://boxyhq.com/docs/jackson/sso-providers).
```

Comment:
do not link to their docs.. it's not clear what the provider name should be. Instead, provide a list of provider-names we support clearly.

Reply:
Updated